core/src/modules/ops/record.ts

```diff
@@ -260,6 +258,7 @@ export namespace RecordOps {
     recordProperties: {
       [key: string]: (string | number | boolean | Date)[] | any;
     }[],
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     toLock = true,
```

Comment:
This argument is not really being used right now, so I'm not sure whether to remove it and update all the instances that use it or leave it.

Reply:
If there's enough of these I'd say icebox a ticket to remove.